Frontend/src/api/registration/delete/delete_registration.js

```diff
@@ -0,0 +1,20 @@
+export default async function delete_registration(competitor_id, competition_id){
+    const formData = new FormData();
+    formData.append('competitor_id', competitor_id);
+    formData.append('competition_id', competition_id);
+
+    try {
```

Comment:
We should extrapolate the try catch into it's own function. We can a `MyFetch` as a function that fetches data from the backend and does the try catch and error formatting.

Reply:
I also don't believe we need a separate directory for each file. These files are already scoped to the registration feature.